oat-sa/tao-core-ui-fe
# [TAO-9478] Add play from here icon

Related to : https://oat-sa.atlassian.net/browse/TAO-9478
 
Add new icon to tao icon font
  
#### How to test
 
- Prepare an instance of tao with taoDevTools extension
- Navigate to TAO Icon Font in the back office application
- Check 'play-from-here' icon
  
#### Dependencies
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-devtools/pull/164
 - [ ] https://github.com/oat-sa/tao-core/pull/2344
